EMIP/1001-2000/EMIP01960.xml

```diff
@@ -55,7 +55,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <msItem xml:id="ms_i3">
                      <locus from="14v" to="88v" facs="019"/>
                      <title type="complete" ref="LIT2393tabiba"/>
-                     <note>Something may wrong in the description, Tabiba tabibab is starting from f. 18v.</note>
+                     <note>Something may be wrong in the description, <ref type="work" corresp="LIT2393tabiba"/> is starting from <locus target="#18v"/>.</note>
```

Comment:
I think you should at least tag @Ralph-Lee-UK so that he can (1) see the changes of the ID and (2) check the description

Reply:
PS I also am a bit confused as these changes seem to already have been merged

Reply:
is there anything new in comparison to https://github.com/BetaMasaheft/Manuscripts/pull/796?
If not, this branch should be simply deleted without merging